ui/app/components/modal-form/oidc-assignment-template.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @module ModalForm::OidcAssignmentTemplate
+ * ModalForm::OidcAssignmentTemplate components are meant to render within a modal for creating a new assignment
```

Comment:
Could update this to say that it wraps the AssignmentForm to create a model for the form component